build.gradle.kts

```diff
@@ -59,9 +59,17 @@ kotlin {
     jvmToolchain(21)
 }
 
+val shadowJarOnly: Boolean = project.property("shadowJarOnly")?.toString()?.toBoolean() ?: false
```

Comment:
I don't anticipate this being something that gets toggled on and off in this plugin, but I wanted this to be something I could move into the template (which actually should in part be a Gradle plugin...)